.buildkite/pipeline.yml

```diff
@@ -163,7 +163,7 @@ steps:
       # Runs only on main and release branches
       - label: "Unit tests - macOS 13"
         command: ".buildkite/scripts/steps/unit-tests.sh"
-        branches: "main 8.* 9.*"
```

Comment:
TODO: revert once we confirm that macOS 13 unit tests run successfully in CI on this PR.